### DISCUSSION TITLE
hobbies/galgame

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#16] [General]
# hobbies/galgame

galgame

https://absx.pages.dev/hobbies/galgame.html

**Comment by Ic-arbon** [comment]
我喜欢你

**Comment by fbkno1intheworld-hub** [comment]
老大可以求一下ちぇ～んじ！～あの娘になってクンクン的资源吗

**Reply by lxl66566** [reply]
[真红小站](https://www.shinnku.com/search?q=ちぇ〜んじ!%20〜あの娘になってクンクンペロペロ〜)有

**Comment by Ic-arbon** [comment]
Hi Xuanle,

抄送给我是故意的还是不小心的

Regards,

Tianyi Deng
________________________________
发件人: Absolutex ***@***.***>
发送时间: 2026年8月23日 10:12
收件人: lxl66566/lxl66566.github.io ***@***.***>
抄送: Ic-arbon ***@***.***>; Comment ***@***.***>
主题: Re: [lxl66566/lxl66566.github.io] hobbies/galgame (Discussion #16)


真红小站<https://www.shinnku.com/search?q=%E3%81%A1%E3%81%87%E3%80%9C%E3%82%93%E3%81%98!%20%E3%80%9C%E3%81%82%E3%81%AE%E5%A8%98%E3%81%AB%E3%81%AA%E3%81%A3%E3%81%A6%E3%82%AF%E3%83%B3%E3%82%AF%E3%83%B3%E3%83%9A%E3%83%AD%E3%83%9A%E3%83%AD%E3%80%9C>有

―
Reply to this email directly, view it on GitHub<https://github.com/lxl66566/lxl66566.github.io/discussions/16?email_source=notifications&email_token=AE35AAOYGUVOOGADNQDDSN35LJHHLA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBRGIYDMNJQUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-18120650>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AE35AAPNTSP42AIDCWPF6FD5LJHHLAVCNFSNUABHKJSXA33TNF2G64TZHM2DQOBUGA3TSOJWHNCGS43DOVZXG2LPNY5TKNZUGUYTQMFBOYBA>.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS<https://github.com/notifications/mobile/ios/AE35AAOMN6JZZ62O2ZOCRWL5LJHHLA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBRGIYDMNJQUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSVGM33PORSXEX3JN5ZQ> and Android<https://github.com/notifications/mobile/android/AE35AAK5NAAPKVKHACPAT3L5LJHHLA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBRGIYDMNJQUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>. Download it today!

**Reply by lxl66566** [reply]
不是我发的，要怪 Github 喵，Notifications -> Unsubscribe 喵
